cli/lib/settings/rebase_data_setting.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
+import 'package:stax/settings/base_settings.dart';
+import 'package:stax/settings/setting.dart';
+
+class RebaseData {
+  final bool hasTheirsFlag;
+  final bool hasOursFlag;
+  final String rebaseOnto;
+  final List<Map<String, String>> branches;
```

Comment:
Let's define `RebaseStep` class, which will hold information about rebase step. So in the future we can put more information into it.
```suggestion
  final List<RebaseStep> steps;
```